### PR TITLE
Final tweak Azure.ResourceManager.Media

### PR DESCRIPTION
**Pull request by ArthurMa1978**

### CODE REVIEW

**Comment by ArthurMa1978** [comment]
Update the sdk for https://github.com/Azure/azure-sdk-for-net/pull/31368

File: sdk/mediaservices/Azure.ResourceManager.Media/api/Azure.ResourceManager.Media.netstandard2.0.cs [hunk 1]
```diff
@@ -256,19 +113,19 @@ public partial class MediaAssetResource : Azure.ResourceManager.ArmResource
...
-        public virtual Azure.AsyncPageable<System.Uri> GetStorageContainerUrisAsync(Azure.ResourceManager.Media.Models.GetStorageContainersContent content, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual Azure.Pageable<Azure.ResourceManager.Media.Models.AssetStreamingLocator> GetStreamingLocators(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual Azure.AsyncPageable<Azure.ResourceManager.Media.Models.AssetStreamingLocator> GetStreamingLocatorsAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Pageable<System.Uri> GetStorageContainerUris(Azure.ResourceManager.Media.Models.StorageContainerSasContent content, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```
Comment (ArcturusZhang) [inline sdk/mediaservices/Azure.ResourceManager.Media/api/Azure.ResourceManager.Media.netstandard2.0.cs:125]: Can we rename this type `StorageContainerSasContent`? storage RP has a similar class

File: sdk/mediaservices/Azure.ResourceManager.Media/api/Azure.ResourceManager.Media.netstandard2.0.cs [hunk 2]
```diff
@@ -290,8 +147,8 @@ public partial class MediaAssetTrackCollection : Azure.ResourceManager.ArmCollec
     public partial class MediaAssetTrackData : Azure.ResourceManager.Models.ResourceData
     {
         public MediaAssetTrackData() { }
-        public Azure.ResourceManager.Media.Models.MediaProvisioningState? ProvisioningState { get { throw null; } }
-        public Azure.ResourceManager.Media.Models.AssetTrackInfo Track { get { throw null; } set { } }
+        public Azure.ResourceManager.Media.Models.MediaServicesProvisioningState? ProvisioningState { get { throw null; } }
+        public Azure.ResourceManager.Media.Models.TrackBase Track { get { throw null; } set { } }
```
Comment (ArcturusZhang) [inline sdk/mediaservices/Azure.ResourceManager.Media/api/Azure.ResourceManager.Media.netstandard2.0.cs:151]: Could this type be `MediaAssetTrackBase`?

File: sdk/mediaservices/Azure.ResourceManager.Media/api/Azure.ResourceManager.Media.netstandard2.0.cs [hunk 3]
```diff
@@ -724,7 +720,7 @@ public partial class StreamingPolicyResource : Azure.ResourceManager.ArmResource
 }
 namespace Azure.ResourceManager.Media.Models
 {
-    public partial class AacAudio : Azure.ResourceManager.Media.Models.AudioCommonProperties
+    public partial class AacAudio : Azure.ResourceManager.Media.Models.AudioBase
```
Comment (ArcturusZhang) [inline sdk/mediaservices/Azure.ResourceManager.Media/api/Azure.ResourceManager.Media.netstandard2.0.cs:723]: Maybe `MeidaAudioBase`?

**Review by ArcturusZhang — APPROVED** [review]
We have quite a few XXBase class whose names are short and might be generic. Feel free to prepend the RP name before them or ignore these comments

File: sdk/mediaservices/Azure.ResourceManager.Media/api/Azure.ResourceManager.Media.netstandard2.0.cs (hunk 1 above)
Comment (ArthurMa1978) [reply sdk/mediaservices/Azure.ResourceManager.Media/api/Azure.ResourceManager.Media.netstandard2.0.cs:125]: Renamed to `MediaAssetStorageContainerSasContent`

**Pull request merged**